src/main/java/org/broadinstitute/hellbender/tools/dataflow/MarkDuplicatesDataflow.java

```diff
@@ -218,20 +269,48 @@ public void processElement(final ProcessContext context) throws Exception {
                             }
 
                             //order by score
-                            final List<PairedEnds> scored = Ordering.natural().reverse().onResultOf((PairedEnds pair) -> pair.score()).immutableSortedCopy(paired.get(true));
+                            List<PairedEnds> scored = Ordering.natural().reverse().onResultOf((PairedEnds pair) -> pair.score()).sortedCopy(paired.get(true));
```

Comment:
prefer  a method reference to a lambda when possible.  `onResultOf((PairedEnds pair) -> pair.score())` -> `onResultOf(PairedEnds::score)`